### PR TITLE
Feat/score username saveload

### PR DESCRIPTION
**Pull request by theophile-jr**
Feature Saveinfo.hpp, allowing to save & load info from file. Score & username

### CODE REVIEW

**Review by Thomaltarix — APPROVED** [review]
Seems fine :+1:

File: src/Core/main.cpp [hunk 1]
```diff
@@ -7,10 +7,10 @@
 
 #include <iostream>
 #include "Arcade.hpp"
+#include "Saveinfo.hpp"
```
Comment (mathieurobert1) [inline src/Core/main.cpp:10]: useless include

**Review by mathieurobert1 — COMMENTED** [review]
Just a useless include but it's fine

**Review by Thomaltarix — APPROVED** [review]
Seems better

**Pull request merged**